truenas_install/__main__.py

```diff
@@ -434,6 +466,21 @@ def main():
 
                 write_progress(0.5, "Performing post-install tasks")
 
+                for entry in TRUENAS_DATASETS:
+                    if not (force_mode := entry.get("mode")):
+                        continue
+
+                    os.chmod(f"{root}/{entry['name']}", force_mode)
```

Comment:
Do we want this to be recursive or the option to be configurable ?

Reply:
Since it's a newly created dataset it won't have contents (unless squashfs already had it - in which case it should probably be fixed in package install).